neuralmonkey/encoders/transformer.py

```diff
@@ -67,6 +71,7 @@ def __init__(self,
                  n_heads: int,
                  dropout_keep_prob: float = 1.0,
                  attention_dropout_keep_prob: float = 1.0,
+                 target_space_id: int = None,
```

Comment:
Nestálo by za to z tohohle humusu udělat konstantu? Má to nějakej výzam mít to nastavitelný jindy než při importu?

Reply:
No tahle konstanta muze byt jakekoliv cislo <0;31>  a navic to ani nevyctes z hparams.

Abys ho spolehlive vytahl, tak bys pri importu musel spoustet t2t binarku (minimalne nejaky dry-run), abys tohle cislo nejak vycucal.

Reply:
Řikal jsem __jindy než při importu__

Reply:
Pokud tam bude konstanta, tak tam zbytecne pridavas matici shape=[32, emb_size]. Kterou potrebujes jen pri tom importu (a nebo pokud si sam chces trenovat "multimodal" modely.